subsys/usb/usb_c/usbc_pe_snk_states.c

```diff
@@ -432,6 +432,11 @@ void pe_snk_ready_run(void *obj)
 			case PD_DATA_SOURCE_CAP:
 				pe_set_state(dev, PE_SNK_EVALUATE_CAPABILITY);
 				break;
+			case PD_DATA_VENDOR_DEF:
+				if (prl_get_rev(dev, PD_PACKET_SOP) > PD_REV20) {
```

Comment:
Can you add a comment indicating that the Zephyr PD implementation doesn't support VDMs.  Per PD 2.0, VDMs are ignored and per PD 3.0  send a not supported message.